study-builder/studies/rgp/study.conf

```diff
@@ -100,7 +100,11 @@
         {
           "type": "ACTIVITY",
           "activityCode": "ENROLLMENT",
-          "expression": """user.studies["RGP"].forms["ENROLLMENT"].hasInstance()"""
+          "expression": """user.studies["RGP"].forms["PREQUAL"].hasInstance() &&
+                           user.studies["RGP"].forms["PREQUAL"].questions["prequal_question"].answers.hasOption("UNDERSTANDS_ENGLISH") &&
+                           user.studies["RGP"].forms["PREQUAL"].questions["prequal_question"].answers.hasOption("LIVES_US") &&
+                           user.studies["RGP"].forms["PREQUAL"].questions["prequal_question"].answers.hasOption("HAS_CONDITION") &&
+                           user.studies["RGP"].forms["PREQUAL"].questions["prequal_question"].answers.hasOption("HAS_DOCTOR")"""
```

Comment:
This makes me think that a nice little feature to add to PEX is a `hasAllOptions()` predicate.

Reply:
For some reason I had it in my head at first that hasOptions was a thing and then realized when I got an error that it wasn't.  So I think this would be a good feature, although not particularly high priority.